third_party/readme.txt

```diff
@@ -1,2 +0,0 @@
-The contents of this folder are from GCC, and are licensed
-under the GPL (see details in the files themselves).
```

Comment:
I think wethird_party/websockify is still under GPL, but there is a license file in that directory so deleting this seems reasonable.